app/js/components/cohorts/actionsComponent.jsx

```diff
@@ -125,11 +130,21 @@ class ActionsComponent extends Component {
 		};
 	}
 
+    deleteCohort(cohortId) {
+        return (e) => {
+            const apiHelper = new ApiHelper();
+            apiHelper.delete(`/cohort/${cohortId}`)
+                .then(res => {
+                    this.getAllCohorts();
```

Comment:
I feel if we aren't using the 'res' argument returned after delete, there might be no need to include it. What do you think about this @andela-foladeji ?